src/sortStudents.ts

```diff
@@ -1,16 +1,51 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+  ag: number
```

Comment:
ag shouldn’t be required. We call sortStudents with students array where isn't have ag

```suggestion
  ag?: number
```